tests/test_device_tracker.py

```
@@ -18,6 +19,70 @@
 pkg = importlib.import_module("custom_components.opnsense")
 
 
+def _make_scanner_entity(
+    coordinator: MagicMock,
+    make_config_entry: Callable[..., MockConfigEntry],
+    *,
+    coordinator_data: Any | None = None,
```

Ruff (0.15.18)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `coordinator_data`

(ANN401)